android/build.gradle.kts

```diff
@@ -119,11 +119,12 @@ dependencies {
         implementation(MATERIAL)
         implementation(NAVIGATION)
         implementation(UI_TOOLING)
-        implementation(COIL_COMPOSE)
+
         implementation(ACCOMPANIST_PAGER)
         implementation(ACCOMPANIST_PAGER_INDICATORS)
         implementation(ACCOMPANIST_PLACEHOLDER)
-
+        implementation(COIL_COMPOSE)
+        implementation(CONSTRAINT_LAYOUT)
```

Comment:
Curious, this plugin is for getting the screen size?

Reply:
@blyscuit [That's a more powerful Layout in Android](https://developer.android.com/jetpack/compose/layouts/constraintlayout) for both XML and Jetpack Compose. It combines the features of Row, Column, Box, etc with constraint attrs. 🤓